### PR TITLE
Initial support for orange pi 3 lts

### PR DESCRIPTION
**Pull request by ludiazv**
Support for opi 3 lts. It does not include support for audio or wifi/bt yet. The patches were ported from a build root distribution that enable basic support for the board.

### CODE REVIEW

File: recipes-kernel/linux/linux-mainline_6.1.9.bb [hunk 1]
```diff
@@ -21,3 +21,31 @@ SRC_URI:append:orange-pi-zero2  = " \
...
+
+
+SRC_URI:append:orange-pi-3lts  = " \
+        file://orangepi-3lts.cfg \
+        file://0001-Input-axp20x-pek-allow-wakeup-after-shutdown.patch \
+        file://0002-clk-Implement-protected-clocks-for-all-OF-clock-prov.patch \
+        file://0003-Revert-clk-qcom-Support-protected-clocks-property.patch \
+        file://0007-rtc-sun6i-Allow-RTC-wakeup-after-shutdown.patch \
+        file://0009-firmware-arm_scpi-Support-unidirectional-mailbox-cha.patch \
+        file://0012-arm64-dts-allwinner-h6-Add-SCPI-protocol.patch \
```
Comment (nandra) [inline recipes-kernel/linux/linux-mainline_6.1.9.bb:33]: Why this numbering please? Can we have 0001 - 00xx. Thanks.

File: recipes-kernel/linux/linux-mainline_6.1.9.bb [hunk 2]
```diff
@@ -21,3 +21,31 @@ SRC_URI:append:orange-pi-zero2  = " \
...
+        file://0043-HACK-h6-Add-HDMI-sound-card.patch \
+        file://0049-arm64-dts-allwinner-h6-Add-AC200-EPHY-related-nodes.patch \
+        file://0053-mmc-sunxi-fix-unusuable-eMMC-on-some-H6-boards-by-di.patch \
+        file://0065-wip-fix-H6-4k-60.patch \
+        file://0066-arm64-dts-allwinner-h6-Fix-Cedrus-IOMMU-again.patch \
+        file://0073-iommu-sun50i-Allow-page-sizes-multiple-of-4096.patch \
+        file://0077-OrangePi-3-LTS-support.patch \
+        file://2000-arm64-dts-allwinner-Enforce-consistent-MMC-numbering.patch \
+"
+#  For wifi and BT these are under testing
```
Comment (nandra) [inline recipes-kernel/linux/linux-mainline_6.1.9.bb:47]: Please remove dead code. We can add it later ;)

**Review by nandra — COMMENTED** [review]
Other then small things looks good. Thanks for your work! :+1:

**Comment by ludiazv** [comment]
Ok. I will do the changes during the week. I will include also a minor change for make USB3 working at full speed.

**Comment by nandra** [comment]
@ludiazv any update on this please? Thanks.

**Comment by ludiazv** [comment]
I have a new version for suppoort for this board with a more.... I forgot to do the PR. I will submit a new PR.

**Comment by nandra** [comment]
Closing.

**Pull request closed without merging**